imports/components/cards/cards.Activity.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import { Component } from "react";
+import styled from "styled-components";
+import moment from "moment";
+import { Link } from "react-router";
+import { Error, Success } from "../icons";
+
+const Paragraph = styled.p`
+  font-family: colfax-web, sans-serif;
+`;
+
+const BoldedDate = styled(Paragraph)`
+  font-weight: bold;
+  display: inline-block;
+  vertical-align: top;
+`;
+
+const Strong = styled.strong`
+  font-family: colfax-web, sans-serif;
+`;
+
+type IActivity = {
+  status: string,
+  transaction: Object,
+};
+
+export default class Activity extends Component {
+  props: IActivity;
+
+  getClasses = () => {
+    const classes = [
+      "card__item",
+      "soft",
+      "text-left",
+      "soft-bottom",
+      "rounded",
+    ];
+    return classes.join(" ");
+  };
+
+  getStyles = (status: string) => {
+    const styles = {
+      color: "#FFFFFF",
+      fontFamily: "colfax-web, sans-serif",
+      backgroundColor: "#242424",
+      fill: "#FFFFFF",
+    };
+
+    if (status === "failed") {
+      styles.backgroundColor = "#BA3841";
```

Comment:
this part can't, however, we need to add them to junction. I'm good with them staying as is for now! @samclaridge these are new colors?